src/components/tutorials-sidebar/index.tsx

```diff
@@ -0,0 +1,89 @@
+import { Fragment } from 'react'
+import VisuallyHidden from '@reach/visually-hidden'
+import { IconChevronLeft16 } from '@hashicorp/flight-icons/svg-react/chevron-left-16'
+import isAbsoluteUrl from 'lib/is-absolute-url'
+import StandaloneLink from 'components/standalone-link'
+import SkipToMainContent from 'components/sidebar/components/sidebar-nav/skip-to-main-content'
+import { SidebarNavLink } from 'components/sidebar/components/sidebar-nav/sidebar-nav-menu-item'
+import {
+  ListItemProps,
+  SectionListProps,
+  SectionTitleProps,
+  TutorialSidebarProps,
+} from './types'
+import s from './tutorials-sidebar.module.css'
+import { HorizontalRule as DocsHorizontalRule } from 'components/sidebar/components/sidebar-nav/sidebar-nav-menu-item'
+
+function TutorialsSidebar({
+  backToLink,
+  children,
+  ariaLabelledBy,
+}: TutorialSidebarProps) {
+  return (
+    <nav className={s.root} aria-labelledby={ariaLabelledBy}>
+      <div className={s.backToLink}>
+        <StandaloneLink
+          href={backToLink.href}
+          text={backToLink.text}
+          icon={<IconChevronLeft16 />}
+          iconPosition="leading"
+          textSize={200}
+        />
+      </div>
+
+      <div className={s.itemsContainer}>
+        <SkipToMainContent />
+        {children}
+      </div>
+    </nav>
+  )
+}
+
+function SectionList({ items }: SectionListProps) {
+  return (
+    <ul className={s.listRoot}>
+      {items.map(({ text, href, isActive, isExternal }: ListItemProps) => {
+        return (
+          <ListItem
+            key={`${text}${href}`}
+            text={text}
+            href={href}
+            isActive={isActive}
+            isExternal={isExternal}
+          />
+        )
+      })}
+    </ul>
+  )
+}
+
+function ListItem({ href, isActive, text, isExternal }: ListItemProps) {
+  const parsedIsExternal =
+    typeof isExternal == 'boolean' ? isExternal : isAbsoluteUrl(href)
+  const hrefOrFullPath = parsedIsExternal ? { href } : { fullPath: href }
+  return <SidebarNavLink item={{ isActive, title: text, ...hrefOrFullPath }} />
+}
+
+function SectionTitle({
+  text,
+  as = 'h2',
+  id,
+  visuallyHidden,
+}: SectionTitleProps) {
+  const Wrapper = visuallyHidden ? VisuallyHidden : Fragment
+  const Elem = as
+  return (
+    <Wrapper>
+      <Elem className={s.sectionTitle} id={id}>
+        {text}
+      </Elem>
+    </Wrapper>
+  )
+}
```

Comment:
Let's chat about this one more on Monday. It's almost there, but it gives too much flexibility for using the incorrect heading level for labeling sections. The levels also need to be updated, because we can't have multiple `h1` elements in a page, and I also expect the sidebar title will never be the most important heading element in any page.

Reply:
@ashleemboyer  Sounds good - I think I confused`<nav />` being classified as [sectioning content](https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Content_categories#sectioning_content) with the concept of the "sectioning root" ... and also, TIL this seems to have been [_supposed_ to be part of HTML5](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements#multiple_h1_elements_on_one_page) but seems to just have not been implemented yet? Fascinating stuff 😅 

Happy to update these however makes sense. I guess since there's really no "section" isolation, the`h1` could become an `h2`, and the `h2`s would become `h3`s, and that might have a stronger guarantee to fit in nicely with the outline of the page as a whole? 🤔

Reply:
☝️ I took a shot at the semantic markup changes, and at reducing the chance of incorrect heading levels:

- `TutorialSidebar` now requires a `title` string prop, which _always_ renders an `h2` into the `<nav />`, and _always_ references it with `aria-labelledby`
- `SectionTitle` now _only_ ever renders an `h3`

Definitely just making an attempt here - looking forward to ironing it out Monday 🙇‍♂️

Reply:
This is good! Here's the reference I was waiting to share today: https://www.w3.org/WAI/tutorials/page-structure/headings/. It has an example of using a single `h1` for the most important heading on the page, and then using `h2`s to give the sidebar a title, and `h3`s to title each section of the sidebar. Nicely done! 👏🏻